locale/Makefile

```diff
@@ -44,22 +38,20 @@ uniq-po:
 		msguniq $$f -o $$f ; \
 	done
 
-tx-pull:
+tx-pull: $(EDITFILES)
 	tx pull -f
 	for f in $(POFILES) ; do \
 		sed -i 's/^\("Project-Id-Version: \).*$$/\1$(DOMAIN) $(VERSION)\\n"/' $$f; \
 	done
-	-git commit -a -m "i18n - extracting new, pulling from tx"
 
+# Extract strings and update the .pot, prepare .edit.po files
 extract-strings:
 	bundle exec rake gettext:find
 
-reset-po:
-	# merging po files is unnecessary when using transifex.com
-	git checkout -- ../locale/*/*po
+# Merge .edit.po into .po
+update-po:
+	bundle exec rake gettext:find
```

Comment:
Mostly because I'm out of better ideas :)

The rake task does three things internally - it a) extracts strings from source files into the .pot, b) updates .edit.po from .pot, c) updates .po from .edit.po

So in extract-strings, its job is to extract strings from source files into .edit.po, and then we call tx pull.  Then in update-po it's to merge .edit.po back into .po.

The .edit.po intermediate file is a new feature of gettext 3.  I considered fighting it, but decided to just adopt it.